docs/manpage.rst

```diff
@@ -387,6 +387,26 @@ It does so by leveraging the selected system's environment modules system.
    This option can also be set using the :envvar:`RFM_UNLOAD_MODULES` environment variable or the :js:attr:`unload_modules` general configuration parameter.
 
 
+.. option:: --unuse-module-path=PATH
```

Comment:
Why not use a single option and use `+/-` to control addition and deletion of paths?

- `--module-path=+/path/to/add`
- `--module-path=-/path/to/remove`

Reply:
If somebody passes it without `+/-` we could completely override the module path.

Reply:
I think that it's better to have only the `-` convension for unusing otherwise the module path should be used.